packages/core/botpress/src/web/components/Tour/index.jsx

```diff
@@ -93,14 +93,14 @@ export default class GuidedTour extends React.Component {
         <p className={style.big}>There's a module for that.</p>
         <p>Whenever there's a piece of functionality that you need for your bot, think modules.</p>
         <hr />
-        <p>You can install modules via this graphical interface or from the command line interface.</p>
+        <p>You can install modules via this graphical interface or from the command-line interface.</p>
         <p>
           If there's no module for the generic feature you are looking for, please consider creating one or contributing
           to an existing module to add that feature. Botpress relies heavily on the community to thrive. Every single
           contribution is needed, even if it's tiny {emojify(':blush:')}!
         </p>
         <p>
-          Installing a module from the CLI is as simple as typing <code>npm install --save botpress-messenger</code>
+          Installing a module from the command-line is as simple as typing <code>npm install botpress-messenger</code>
```

Comment:
Not sure we need to remo --save flag here

Reply:
`--save` is the default in npm now, so it's moot.

Reply:
Oh, OK.